### PR TITLE
Fix minor bugs and version bump

### PR DESCRIPTION
**Pull request by KapilTR**
Minor changes including version change, socket-path for windows, and typo errors.

### CODE REVIEW

File: modules/contract/contracts.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ class Contracts {
     async getEntryPoints(args) {
         Logger.verbose(`Command : tezster extract-entry-points ${args}`);
         if (args.length < 1) {
-            Logger.warn('Incorrect usage - tezster list-entry-points <contract-absolute-path>');
+            Logger.warn('Incorrect usage - tezster extract-entry-points <contract-absolute-path>');
```
Comment (jvtezsure) [inline modules/contract/contracts.js]: why are you changing command name?

File: modules/setup/setup.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const Docker = require('dockerode');
-var docker = new Docker({ socketPath: '/var/run/docker.sock' });
+var docker = new Docker({ socketPath: '/var/run/docker.sock', hosts: 'tcp://0.0.0.0:2376' });
```
Comment (jvtezsure) [inline modules/setup/setup.js:2]: did you test on linux after these change of adding 'hosts'?

**Review by jvtezsure — CHANGES_REQUESTED** [review]

File: modules/contract/contracts.js (hunk 1 above)
Comment (KapilTR) [reply modules/contract/contracts.js]: the name is not being changed that list just got mistakenly typed insted of extract there.

File: modules/setup/setup.js (hunk 2 above)
Comment (KapilTR) [reply modules/setup/setup.js:2]: yes, I tested on linux, that is working

File: modules/contract/contracts.js (hunk 1 above)
Comment (jvtezsure) [reply modules/contract/contracts.js]: I think list-entry-points is better than extract.

Comment (KapilTR) [reply modules/contract/contracts.js]: yeah have done change.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tezster",
-  "version": "0.2.1",
+  "version": "0.2.2-beta",
```
Comment (jvtezsure) [inline package.json:3]: version change should always happen on dev or master. 
For this CL it's fine. Next time, you need to be mindful about this.

**Review by jvtezsure — APPROVED** [review]

**Pull request merged**